src/convertToObject.js

```diff
@@ -6,7 +6,25 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const resultObject = {};
+  let key = '';
+  let value = '';
+
+  const splitAllstring = sourceString.split(';');
```

Comment:
The variable name 'splitAllstring' does not follow camelCase naming conventions and can be confusing. It would be better named as 'splitAllString' or 'styleDeclarations'.